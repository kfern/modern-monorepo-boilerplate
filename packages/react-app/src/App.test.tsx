// import React from 'react';

import { shallow } from 'enzyme';

import Fixture from './App.fixture';

describe('App', () => {

  it('shallow: Test fixture is stable', () => {
    const wrapper = shallow(Fixture.Test);
    expect(wrapper).toMatchSnapshot();

  });

});
