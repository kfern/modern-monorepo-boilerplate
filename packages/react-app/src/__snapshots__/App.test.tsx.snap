// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App shallow: Test fixture is stable 1`] = `
<Admin
  dataProvider={
    Object {
      "create": [Function],
      "delete": [Function],
      "deleteMany": [Function],
      "getList": [Function],
      "getMany": [Function],
      "getManyReference": [Function],
      "getOne": [Function],
      "update": [Function],
      "updateMany": [Function],
    }
  }
>
  <Resource
    list={[Function]}
    name="posts"
  />
  <Resource
    list={[Function]}
    name="comments"
  />
</Admin>
`;
